demos/http/http_demo_s3_download/http_demo_s3_download.c

```diff
@@ -73,6 +73,11 @@
     #define USER_BUFFER_LENGTH    ( 4096 )
 #endif
 
+/* Pointer to the data to upload.*/
+#ifndef DEMO_HTTP_UPLOAD_DATA
+    #define DEMO_HTTP_UPLOAD_DATA    "Hello World!"
```

Comment:
The demo can be put the upload object path in here. open the file path, read the file and upload to S3. This seems to me is create a file and the content is Hello World.